### PR TITLE
Feature/change membership service to adhere to known agent limitations

### PR DESCRIPTION
**Pull request by orazefabian**
This PR changes the `MembershipService` to adhere to the known limitations of the agent:

1. Usage of the `EntityFramwork` is discarded - the service now uses plain ADO .NET 
2. The request data is not sent as json via the request body, but rather as form parameters using the contenty type: `application/x-www-form-urlencoded`

### CODE REVIEW

File: src/membership-service/Startup.cs [hunk 1]
```diff
@@ -1,69 +1,34 @@
...
-                    options => options.EnableRetryOnFailure(
-                        maxRetryCount: 10,
-                        maxRetryDelay: System.TimeSpan.FromSeconds(60),
-                        errorNumbersToAdd: null)
-                )
-            );
-
-            services.AddScoped<MembershipDbService, MembershipDbService>();
-
+            services.AddCors(options =>
```
Comment (W3D3) [inline src/membership-service/Startup.cs:20]: [DISCUSS] I think the CORS stuff is not needed? Looks like something you forgot to remove after debugging?

File: src/membership-service/Controllers/MembershipController.cs [hunk 2]
```diff
@@ -1,65 +1,122 @@
...
       {
-        result = new MembershipEntity(userid, "FREE");
+        _logger.LogInformation($"GET requested => userid: {userid}");
+
+        await connection.OpenAsync();
+
+        string selectQuery = "SELECT membership FROM membership WHERE userid = " + userid;
+
+
+        _logger.LogInformation($"Executing query: {selectQuery}");
```
Comment (W3D3) [inline src/membership-service/Controllers/MembershipController.cs:40]: [Q] What logging format do we get here?

**Review by W3D3 — APPROVED** [review]
Just some minor things. Fine to merge IMO.

File: src/membership-service/Startup.cs (hunk 1 above)
Comment (orazefabian) [reply src/membership-service/Startup.cs:20]: Yeah, forgot to remove this

File: src/membership-service/Controllers/MembershipController.cs (hunk 2 above)
Comment (W3D3) [reply src/membership-service/Controllers/MembershipController.cs:39]: Would have preferred it to look like the other services but changing this should be a follow up.

**Review by W3D3 — APPROVED** [review]

**Pull request merged**